### PR TITLE
feat: Add Graphic Filter

### PR DESCRIPTION
**Pull request by picheli20**
I'm not quite used to the "react way" -  please let me know if something is not matching the best pratices.

**Motivation**: Sometimes is quite hard to see how's the progress looking all the 3 cases - for example,  looking the graphic bellow is quite hard to see the progress with the cured cases:

Screenshot 2020-05-02 at 15 26 57

But if we are able to isolate and show only cured, we can see that it's growing :)

Screenshot 2020-05-02 at 15 27 07

### CODE REVIEW

File: src/components/DailyGrowth/DailyGrowth.jsx [hunk 1]
```diff
@@ -172,30 +224,35 @@ export default function DailyGrowth() {
...
+                      fill={item.color}
+                    />
+                  ))}
                 </BarChart>
               </ResponsiveContainer>
             </StyledBody>
           </ResponsiveTab>
         </Tabs>
       )}
+      <div className='daily-growth-filter'>
```
Comment (konradkalemba) [inline src/components/DailyGrowth/DailyGrowth.jsx:242]: We're using Styletron and the styles defined for this element should be written like that:
```suggestion
      <div
        className={css({
          display: flex,
          justify-content: space-evenly,
        })}
      >
```
Note: `css` is not defined as for now. You should add its definition in line 79:
```js
  const [css, theme] = useStyletron();
```

File: src/components/DailyGrowth/DailyGrowth.jsx [hunk 2]
```diff
@@ -172,30 +224,35 @@ export default function DailyGrowth() {
...
+          <label>
+            <input
+              name={item.key}
+              type='checkbox'
+              dataKey={item.key}
+              checked={item.selected}
+              onChange={() => handleInputChange(item, index)}
+            />
+            {item.key}
+          </label>
```
Comment (konradkalemba) [inline src/components/DailyGrowth/DailyGrowth.jsx:253]: There is a [Checkbox ](https://baseweb.design/components/checkbox/) component in the UI library we're using. It'll look better imho. :)

File: src/index.css [hunk 3]
```diff
@@ -72,6 +72,11 @@ svg.marker text {
   font-weight: 500;
 }
 
+.daily-growth-filter {
+  display: flex;
+  justify-content: space-evenly;
+}
```
Comment (konradkalemba) [inline src/index.css:78]: It should be removed as stated in [a comment above](https://github.com/konradkalemba/korona.ws/pull/75/files#r419251881).
```suggestion
```

**Comment by picheli20** [comment]
@konradkalemba awesome :) Just fixed it.

**Pull request merged**